public/_redirects

```diff
@@ -180,4 +180,6 @@
 
 /*/studio /:splat/deprecated-software/#ethereum-studio 301!
 
-/en/stablecoins /stablecoins 301!
\ No newline at end of file
+/en/stablecoins /stablecoins 301!
+
+/*/community/events https://esp.ethereum.foundation/devcon-grants 301!
```

Comment:
Don't need the changes in this file. We still want people to land on the community events page on ethereum.org without redirecting. Just looking for the banner.